File: sample/rust/test_commented.rs
fn main() {
    // Already commented out debug statements
    println!("debug: this is commented");
    eprintln!("DEBUG: error message");
    dbg!(variable);

    println!("Active debug message");

    println!("debug: another commented line");

    let x = 10;

    // Multiline commented

    compute();
}

fn compute() {
    eprintln!("debug: computing");
    let result = 42;
    println!("debug: result is {}", result);
}

File: sample/java/TestCommented.java
public class TestCommented {
    public static void main(String[] args) {
        // Already commented out debug statements
        // System.out.println("debug: this is commented");
        // System.err.println("DEBUG: error message");

        System.out.println("Active debug message");

        // System.out.println("debug: another commented line");

        int x = 10;
        // System.out.printf("debug: x = %d\n", x);

        // Multiline commented
        // System.out.println(
        //     "debug: multiline"
        // );

        compute();
    }

    static void compute() {
        // System.err.println("debug: computing");
        int result = 42;
        System.out.println("debug: result is " + result);
    }
}
